src/quo/components/list_items/dapp/view.cljs

```diff
@@ -25,14 +25,16 @@
         :style  {:width 32 :height 32}}]
       [rn/view {:style style/user-info}
        [text/text
-        {:weight :semi-bold
-         :size   :paragraph-1
-         :style  (style/style-text-name theme)}
+        {:weight          :semi-bold
+         :size            :paragraph-1
+         :number-of-lines 1
```

Comment:
I'm assuming that this component `list_items/dapp` is not being used anywhere else. Is that correct?

Reply:
Yes.
In the code you can also see it being used in `status-im.contexts.wallet.account.tabs.dapps.view` ns but it's in fact not used there — the list of dapps on that tab is hardcoded to `[]`